src/ess/loki/workflow.py

```diff
@@ -0,0 +1,25 @@
+# This file is used by beamlime to create a workflow for the Loki instrument.
+# The function `live_workflow` is registered as the entry point for the workflow.
+import numpy as np
+import scipp as sc
+from beamlime import WorkflowResult
+from scippneutron.io.nexus.load_nexus import JSONGroup
+
+
+def live_workflow(group: JSONGroup) -> WorkflowResult:
+    """Example live workflow function for Loki.
+
+    This function is used to process data with beamlime workflows.
+    """
+    # We do not consume the incoming data right now.
+    _ = group
+    return {
+        'IofQ': sc.DataArray(
+            data=sc.array(dims=['Q'], values=np.zeros(101)),
```

Comment:
This should actually be length 100, since Q is typically bin-edges in our applications.

Reply:
Note there is also `sc.zeros`, which avoids Numpy in this case.